tools/delete_candidate_info.php

```diff
@@ -304,29 +312,35 @@ function deleteTimepoint($sessionID, $confirm) {
             'SELECT * FROM ' . $DB->escape($instrument['Test_name']) . ' WHERE CommentID=:cid',
             array('cid' => $instrument['CommentID'])
         );
+        echo "$instrument\n";
```

Comment:
Should be $instrument['Test_name'], otherwise we get this error:  PHP Notice:  Array to string conversion in /var/www/loris/tools/delete_candidate_info.php on line 315

but most importantly, we don't get the name of the test!